pandas/tests/series/test_reductions.py

```diff
@@ -9,6 +9,13 @@
 import pandas._testing as tm
 
 
+def test_reductions_series_strings():
+    # GH#31746
+    ser = Series(["a", "b"], dtype="string")
+    assert ser.max() == "b"
```

Comment:
Could you use `pytest.mark.parameterize` over `min/max` and the expected result?

Reply:
Of course!